lib/AppSamples/ReactiveForms/view/contactUs.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+void contact(
```

Comment:
add proper comments for class , function , import of any libraries and usage to it